huxley/www/js/components/AdvisorFeedbackView.js

```diff
@@ -0,0 +1,196 @@
+    this._delegatesToken && this._delegatesToken.remove();
+    this._assignmentsToken && this._assignmentsToken.remove();
+  },
+
+  render: function() {
+    return (
+      <InnerView>
+        <h2>Feedback</h2>
+        <p>
+          Here you can view chairs feedback on your delegates, as well as their attendance.
+        </p>
+          <div className="table-container">
+            <table className="table highlight-cells">
+              <thead>
+                <tr>
+                  <th>Committee</th>
+                  <th>Delegates</th>
+                  <th>1</th>
+                  <th>2</th>
+                  <th>3</th>
+                  <th>4</th>
+                  <th>Summary</th>
+                </tr>
+              </thead>
+              <tbody>
+                {this.renderAssignmentRows()}
+              </tbody>
+            </table>
+          </div>
+      </InnerView>
+    );
+  },
+
+  renderAssignmentRows: function() {
+    var assignments = this.state.assignments
+    var committees = this.state.committees;
+    var countries = this.state.countries;
+    var feedback = this.state.feedback;
+    return this.state.assignments.map(function(assignment) {
+      var delegates = feedback[assignment.id];
+      if (delegates == null) {
+        return;
+      }
+      return (
+        <tr>
+          <td>{committees[assignment.committee].name}</td>
+          <td>{countries[assignment.country].name}
+          </td>
+          <td>
+            <input
+              className="choice"
+              type="checkbox"
+              checked={delegates.session_one}
+              disabled
+            />
+          </td>
+          <td>
+            <input
+              className="choice"
+              type="checkbox"
+              checked={delegates.session_two}
+              disabled
+            />
+          </td>
+          <td>
+            <input
+              className="choice"
+              type="checkbox"
+              checked={delegates.session_three}
+              disabled
+            />
+          </td>
+          <td>
+            <input
+              className="choice"
+              type="checkbox"
+              checked={delegates.session_four}
+              disabled
+            />
+          </td>
+          <textarea
+              className="text-input"
+              style={{"width": "95%"}}
+              defaultValue={delegates.published_summary}
+              disabled
+            />
+        </tr>
+      )
+    }.bind(this));
+  },
+
+
+  /*
+    To make it easier to assign and unassign delegates to assignments we maintain
+    a state variable called "feedback". "feedback" is a dictionary whose key is
+    an assignment id, and value is an array representing slots for two delegates
+    to be feedback to it. This way we can easily manage the relationship from
+    assignment to delegates via this object.
+  */
+
+  prepareFeedback: function(delegates, assignments) {
```

Comment:
you don't use the `assignments` variable in this function, so you should remove it